Sources/Workspace/Workspace.swift

```diff
@@ -941,6 +947,17 @@ public class Workspace {
                 case .checkout(let checkoutState):
                     packagePath = checkoutsPath.appending(managedDependency.subpath)
                     version = checkoutState.version
+                    
+                    // In case the checkouts folder for the dependency was deleted,
+                    // clone it again.
+                    if !fileSystem.isDirectory(packagePath) {
```

Comment:
Do you suggest that the new `validateManagedDependencies` function call `clone` to fix the paths? Should it be called something like `fixManagedDependencies` or `updateManagedDependencies` to make it clear its not just checking states?